README.md

```diff
@@ -58,6 +56,12 @@ If you're creating a new topic, keep the following in mind as well:
 | reference | Any reference-type article that isn't auto-generated. |
 | article | Use for community-contributed content (that is, anything from outside the engineering team or the content team at Microsoft. |
 
+## Merging to the live branch
+
+Periodically, about once a month, a manual RI is performance from `main` -> `live` branch.
```

Comment:
I thought Jon told us we can merge whenever we want, and we now have permissions to do so? Who is doing this monthly and what criteria are we using for it?

Reply:
I don't think these 2 lines are negating any of that. 

Usually, things aren't urgent and the monthly docs tend to be done by @JonDouglas most frequently, but have been done by others: https://github.com/NuGet/docs.microsoft.com-nuget/pulls?q=is%3Apr+live+is%3Aclosed+. 

What we're trying to communicate here is that the one-off contributor doesn't have to bother if they don't want to, as it'll be pushed to live eventually.